sensor/common/networkflow/manager/manager_impl.go

```diff
@@ -530,7 +530,7 @@ func (m *networkFlowManager) enrichHostContainerEndpoints(hostConns *hostConnect
 		m.enrichContainerEndpoint(&ep, status, enrichedEndpoints)
 		// If processes listening on ports is enabled, it has to be used there as well before being deleted.
 		used := status.used && (status.usedProcess || !env.ProcessesListeningOnPort.BooleanSetting())
-		if used && status.lastSeen != timestamp.InfiniteFuture {
```

Comment:
It looks like we have been marking endpoints with `rotten=true` but not deleting them. I'm assuming that was an oversight in this [PR](https://github.com/stackrox/rox/pull/8425).

Reply:
Yes. It does look like an oversight from that PR.